crates/sqlexec/src/planner/physical_plan/copy_to.rs

```diff
@@ -161,7 +157,7 @@ impl CopyToExec {
         };
 
         let stream = execute_stream(self.source, context.clone())?;
-        let count = sink.write_all(vec![stream], &context).await?;
+        let count = sink.write_all(stream, &context).await?;
```

Comment:
this is maybe actually a big deal? wouldn't this mean that we don't have to collect the entire stream in memory to write it?

Reply:
Nah this is effectively the same as before. The interface changed from accepting a vector of multiple streams with the implementation being expected to read from each, to just accepting a single stream. I'm not sure why the change was made (and before it was changed to accepting a vector, it previously just accepted a single stream).